.vscode/settings.json

```diff
@@ -1,3 +1,8 @@
 {
-    "typescript.tsdk": "node_modules/typescript/lib"
+  "typescript.tsdk": "node_modules/typescript/lib",
+  "editor.formatOnSave": false,
```

Comment:
Can we let the user decide? For instance, I prefer formatting the code when I save the file, rather than when I copy/paste it from another source.

Reply:
In addition, in fact, this configuration is that once you save the code, it will be automatically formatted. The effective configuration is the last line `"source.fixAll.eslint": "always"`

Reply:
Yes, we don't use Prettier in our project, all formatting is done by eslint. You should not run Prettier here.

The code formatting will happen automatically before every commit. But you can run it manually for all files with `yarn lint:eslint:fix` if needed.

My point was that not all users want to opt-out `editor.formatOnSave` option. We should let them decide when to run the formatter.